wp-content/plugins/hurumap/api.php

```diff
@@ -10,6 +10,14 @@ function register_routes()
             'callback'              => 'get_charts'
         ),
     ));
+    //single hurumap
+    $endpoint_hurumap_chart = '/hurumap/(?P<chart_id>[\w\-]+)$';
```

Comment:
Shouldn't we reuse `/charts`?

Reply:
yes, this is not used at all. I will remove it